### ISSUE TITLE
Support cmdheight=0

### ISSUE THREAD

**Issue by jemag** [hrsh7th/vim-searchx#11]
First, just wanted to say that I love this plugin, thank you for it.

With the newly added cmdheight=0, the normal search works fine (adds a line to show the text we are searching for) :
![image](https://user-images.githubusercontent.com/7985687/173409019-1b659ae5-15c3-4257-8b0f-d2bb02adfa28.png)

However, with this plugin, as soon as we start typing the search item, this bar will disappear and it basically starts typing behind the statusline:
![image](https://user-images.githubusercontent.com/7985687/173408915-f2f3c2db-dd40-4311-97a8-7c64c0579a2f.png)
So basically, we cannot see what we are currently typing

**Comment by jemag** [comment]
Might actually be an upstream problem, not quite sure. Some issues are popping up : 
https://github.com/neovim/neovim/issues/18958
https://github.com/neovim/neovim/issues/18955

**Comment by Shougo** [comment]
Please test https://github.com/neovim/neovim/pull/18961

**Comment by jemag** [comment]
@Shougo  thanks a lot, it works! Will close since this is upstream related and will be fixed once PR is merged
